extern/acts/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# CMake include(s).
+cmake_minimum_required( VERSION 3.11 )
+include( FetchContent )
+
+# Tell the user what's happening.
+message( STATUS "Building Acts as part of the TRACCC project" )
+
+# Declare where to get Acts from.
+set( TRACCC_ACTS_SOURCE
+   "URL;https://github.com/krasznaa/acts/archive/refs/tags/v10.0.1.tar.gz;URL_MD5;fcd2f2f7b5810e34b8faa12634043be3"
```

Comment:
Is there a reason we use version 10.0.1 specifically? I know Acts is up to 14.1.0, is this the last working version, or did we pick this arbitrarily?

Reply:
I wrote about this in the description of the PR... Currently traccc is set up to build v10.0.0. I tried building the current seed-finding code against v14.1.0, but it didn't. We would need to update the code in this repository to make it compatible with the latest version of Acts.

I was not going to do that in this PR... Once https://github.com/acts-project/acts/pull/1071 is accepted, and a new Acts release is made, we can update to that in a separate PR.

Reply:
Sure, I didn't really catch that from the PR description, but that makes sense (upgrading should be on the to-do list!).